### DISCUSSION TITLE
官方教程中的FE idea调试环境搭建，不清晰，thrift这一步该怎么操作呢？

### DISCUSSION THREAD

**Discussion by whutpencil** [apache/doris#7118] [A - General / Q&A] [answered]
![image](https://user-images.githubusercontent.com/24907215/141718985-ab48565d-7549-468c-a4a3-64b314d3b0f8.png)

这一步骤能详细一点吗，不知道该如何操作？

**Comment by hf200012** [comment, marked as answer]
You can compile fe in the docker environment, and then copy the generated java files in the fe directory to the directory corresponding to your development environment，
The following two directories：
fe-common/target/generated-sources/
fe-core/target/generated-sources/
